src/jwt.mli

```diff
@@ -199,17 +192,19 @@ val find_claim :
   payload ->
   string
 
+[@@@warning "-32"]
 val string_of_payload :
   string ->
   payload
+[@@@warning "-32"]
```

Comment:
Did you mean `+32`?

Maybe better just to remove this duplicate definition rather than suppress the warning.